src/io/PinWriter.cpp

```diff
@@ -158,6 +185,28 @@ void PinWriter::write(MatchCollection* collection, string database) {
   setEnabledStatus("NegLog10CombinePValue", combine_p);
   setEnabledStatus("TailorScore", tailor);
 
+  // DIAmeter related, added by Yang
+  if (!MathUtil::AlmostEqual(Params::GetDouble("coeff-precursor"), 0)) {
+     setEnabledStatus("PrecursorIntRankM0", collection->getScoredType(PRECURSOR_INTENSITY_RANK_M0));
```

Comment:
I can't remember. But did you describe the different between M0, M1, and M2. You should describe at least once somewhere

Reply:
Yes, it is described in the header of output file